### ISSUE TITLE
How to get the entity embeddings

### ISSUE THREAD

**Issue by SRL94** [berlincho/RGCN-pytorch#1]
After training the R-GCN on a dataset, how do I get the entity embeddings?

**Comment by SRL94** [comment]
Is it the output (x) of forward function in model.py?

**Comment by berlincho** [comment]
Yes, the output of forward function is the node(entity) embedding of the input graph.
